torch/quantization/make_modules.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
This file is just a utility to generate the modules easier -- in case we need to generate a batch of modules at once

Reply:
Put this file into something more descriptive (such that it's not a normal module that can be imported) or better yet - move tools/build where other codegen recides